### PR TITLE
🚀 feat: add arial label to input component

### PR DESCRIPTION
**Pull request by nataliosilveiraext**
[JIRA Issue](https://gympass.atlassian.net/jira/software/c/projects/ACS/boards/1348?assignee=712020%3A4448bab7-b24c-4e4b-bd76-7d9e3e1c3b36&selectedIssue=ACS-56)

## Description 📄

[Enter the description of your changes]

updated the input to receive aria label correctly, making the additional text be read correctly

## Platforms 📲

- [X] Web
- [ ] Mobile

## Type of change 🔍

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [X] Unit Test
- [X] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

[Upload your screenshots here]

| Before | After |
| ------ | ----- |
|        |       |

### CODE REVIEW

**Review by fabiorgreis — APPROVED** [review]

**Review by gonzagajaquee — APPROVED** [review]

**Review by danielcti — APPROVED** [review]

File: packages/yoga/src/Input/web/Input.test.jsx [hunk 1]
```diff
@@ -204,7 +204,42 @@ describe('<Input />', () => {
         </ThemeProvider>,
       );
 
+      debug();
```
Comment (julianosbentowellhub) [inline packages/yoga/src/Input/web/Input.test.jsx:207]: ❓ Doubt | @nataliosilveiraext 

I think that you forgot this `debug` here.

File: packages/yoga/src/Input/web/Input.test.jsx [hunk 2]
```diff
@@ -204,7 +204,42 @@ describe('<Input />', () => {
...
+
+    it('should have aria-label', () => {
+      const value = 'aria label value';
+      const { debug, getByTestId } = render(
+        <ThemeProvider>
+          <Input label="Input" value="foo" ariaLabel={value} />
+        </ThemeProvider>,
+      );
+
+      debug();
```
Comment (julianosbentowellhub) [inline packages/yoga/src/Input/web/Input.test.jsx:220]: ❓ Doubt | @nataliosilveiraext 

I think that you forgot this `debug` here.

File: packages/yoga/src/Input/web/Input.test.jsx [hunk 3]
```diff
@@ -204,7 +204,42 @@ describe('<Input />', () => {
...
+    });
+    it('should have label value', () => {
+      const value = 'label value';
+      const { debug, getByTestId } = render(
+        <ThemeProvider>
+          <Input label={value} value="foo" />
+        </ThemeProvider>,
+      );
+
+      debug();
```
Comment (julianosbentowellhub) [inline packages/yoga/src/Input/web/Input.test.jsx:236]: ❓ Doubt | @nataliosilveiraext 

I think that you forgot this `debug` here.

**Review by julianosbentowellhub — APPROVED** [review]

File: packages/yoga/src/Input/web/Input.test.jsx (hunk 1 above)
Comment (nataliosilveiraext) [reply packages/yoga/src/Input/web/Input.test.jsx]: was forgotten, I will remove it

File: packages/yoga/src/Input/web/Input.test.jsx (hunk 2 above)
Comment (nataliosilveiraext) [reply packages/yoga/src/Input/web/Input.test.jsx]: was forgotten, I will remove it

File: packages/yoga/src/Input/web/Input.test.jsx (hunk 3 above)
Comment (nataliosilveiraext) [reply packages/yoga/src/Input/web/Input.test.jsx]: was forgotten, I will remove it

**Review by alvim-wh — APPROVED** [review]

**Pull request merged**